core/spec/models/spree/calculator/shipping/flat_percent_item_total_spec.rb

```diff
@@ -8,11 +8,15 @@ module Calculator::Shipping
       let(:variant1) { build(:variant, price: 10.11) }
       let(:variant2) { build(:variant, price: 20.2222) }
 
-      let(:line_item1) { build(:line_item, variant: variant1) }
-      let(:line_item2) { build(:line_item, variant: variant2) }
+      let(:inventory_unit1) { build(:inventory_unit, quantity: 2, variant: variant1, line_item: line_item1) }
```

Comment:
stock_factory using `variants_contents` is creating inventory_units linked to new line_items, so i had to create them before 

https://github.com/spree/spree/blob/main/core/lib/spree/testing_support/factories/stock_factory.rb#L23